test/integration/targets/prepare_nxos_tests/tasks/main.yml

```diff
@@ -1,6 +1,30 @@
+---
+# NXAPI is enabled differently on NX-OS platforms.
```

Comment:
@gundalow Very likely.  I have not investigated this in depth but when we toggle the `sandbox` property it does not seem to have the desired effect.  For now, I would like to go ahead and get this change committed so that our tests work across platforms and then open a separate PR to fix `nxos_nxapi`.